src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -241,7 +241,9 @@ public partial class EventSource : IDisposable
     {
 
 #if FEATURE_EVENTSOURCE_XPLAT
+#pragma warning disable CA1823 // field is used to keep listener alive
         private static readonly EventListener? persistent_Xplat_Listener = XplatEventLogger.InitializePersistentListener();
+#pragma warning restore SA1617
```

Comment:
@stephentoub only a nit: shouldn't this be `#pragma warning restore CA1823` ? Sorry if this isn't important.

Reply:
Good catch!  I'll fix it in my next PR. Thanks.